src/main/java/domain/money/PlayersMoney.java

```diff
@@ -0,0 +1,38 @@
+package domain.money;
+
+import domain.user.Player;
+import java.util.Collections;
+import java.util.Map;
+
+public class PlayersMoney {
+    public static final int DEALER_MULTIPLIER = -1;
+    private final Map<Player, Money> playersMoney;
+
+    public PlayersMoney(Map<Player, Money> playersMoney) {
+        this.playersMoney = playersMoney;
+    }
+
+    public void changeIfBlackjack() {
+        playersMoney.keySet()
+                .forEach(this::changeMoneyIfBlackjack);
+    }
+
+    private void changeMoneyIfBlackjack(Player player) {
+        if (player.isBlackJack()) {
+            playersMoney.replace(player, playersMoney.get(player).changeByBlackJack());
+        }
+    }
+
+    public void changeByPlayerResults(Map<Player, GameResult> playerResults) {
+        playersMoney.forEach(
+                (player, betAmount) -> playersMoney.replace(player, betAmount.change(playerResults.get(player))));
+    }
+
+    public int calculateDealerMoney() {
+        return playersMoney.values().stream().mapToInt(Money::value).sum() * DEALER_MULTIPLIER;
```

Comment:
```suggestion
        int playersMoney = playersMoney.values()
            .stream()
            .mapToInt(Money::value)
            .sum()
        return playersMoney * DEALER_MULTIPLIER;
```
stream을 포현할때 한줄에 점을 여러개 찍지 않아요.

Reply:
더 꼼꼼하게 확인하겠습니다!!